src/client/components/localisation/skybox/model.ts

```diff
@@ -0,0 +1,48 @@
+import { observable } from 'mobx'
+import { BackSide } from 'three'
+import { UniformsUtils } from 'three'
+import { ShaderMaterial } from 'three'
+import { SphereBufferGeometry } from 'three'
+import { Mesh } from 'three'
```

Comment:
An import from `three` should be a view model concern.

The model should should be unaware of how it will be used or displayed, and as such, only contain _raw_ data if possible, and the view model should be concerned with creating library-specific data structures and displaying the data (three/plotly/smoothie, etc). Its not always possible for performance reasons, e.g. when mass real-time data is involved. But in this case, most of these should be able to be moved into the view model without too much trouble 🙂 

So I think the only data that should live in this model is the uniform value settings, since they are just numbers.